test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -145,6 +145,10 @@ class SkipTest(Exception):
     'num_moms': 1,
     'num_comms': 1,
     'num_clients': 1,
+    'min_mom_ram' : 500,
+    'min_mom_disk' : 500,
+    'min_server_ram' : 500,
+    'min_server_disk' : 500,
```

Comment:
Don't you think 500MB of the disk is too less? At least we should use 2Gb or 5Gb...